controllers/workspace/util_test.go

```diff
@@ -58,6 +58,11 @@ func createDevWorkspace(name, fromFile string) {
 	}, 10*time.Second, 250*time.Millisecond).Should(BeTrue())
 }
 
+func updateDevWorkspace(devWorkspace *dw.DevWorkspace) {
```

Comment:
If you keep this function and update it to use a timeout, you might also want to consider taking in an error message string that will be given if the update to the devworkspace fails.